### ISSUE TITLE
tensorflow-1-public/C1/W2/assignment/C1_W2_Assignment_Solution.ipynb  9 Epochs vs 10

### ISSUE THREAD

**Issue by gladwig2** [https-deeplearning-ai/tensorflow-1-public#12]
In the assignment, there are these instructions:

_```
Some notes:

It should succeed in less than 10 epochs, so it is okay to change epochs to 10, but nothing larger
When it reaches 99% or greater it should print out the string "Reached 99% accuracy so cancelling training!"
If you add any additional variables, make sure you use the same names as the ones used in the class
```_

This should be changed to 9 epochs. (also need to account for changing to loss vs accuracy...)

The grader checks to see if you run more than 9 epochs and fails the notebook if you have.

While the callback might prevent you from running the full 10, it does not always work. This is in part because the version of the code I was using (from the live course last week) did not have the warning about if logs.get('accuracy') and I was using logs.get('loss') < 0.01,, which is not the same as accuracy...

Note, any changes here need to be tested with the grader.
